### PR TITLE
websocket: migrate to scripthashes

### PR DESCRIPTION
**Pull request by SomberNight**
fixes https://github.com/spesmilo/electrum/issues/3670

### CODE REVIEW

File: lib/network.py [hunk 1]
```diff
@@ -604,6 +604,9 @@ def process_responses(self, interface):
                 elif method == 'blockchain.scripthash.subscribe':
                     response['params'] = [params[0]]  # addr
                     response['result'] = params[1]
+                elif method == 'blockchain.address.subscribe':
```
Comment (bauerj) [inline lib/network.py]: Why is this needed?

Comment (SomberNight) [reply lib/network.py]: I guess the other parts of this commit make it redundant. That part missing/changed was the reason it wasn't working in the first place, but then I decided to just migrate it :P

However, if other parts of the code still use `blockchain.address.subscribe` then it is needed.
/me goes off looking after an example...
such as the `notify` command
https://github.com/spesmilo/electrum/blob/954897c281eecb45b4f690919b4f623aca57ac28/lib/commands.py#L665

Comment (SomberNight) [reply lib/network.py]: So... possibly related? https://github.com/spesmilo/electrum/issues/3437

Comment (bauerj) [reply lib/network.py]: Would it make sense to switch the other parts to scripthash too instead?

Comment (SomberNight) [reply lib/network.py]: Yes, of course. Still, case in point :D

Comment (SomberNight) [reply lib/network.py]: Ok, I've migrated and fixed the notify command in https://github.com/spesmilo/electrum/commit/d808bf057e21ef728e71f926523ad58056f8c526
Do you think I shoul delete this part of the change?

Comment (bauerj) [reply lib/network.py]: Yes, we should try to remove all traces from the client before the server drops support 👍

Comment (ser) [reply lib/network.py]: Amazing, guys :) I can continue work on the websockets javascript client, then :+1:

**Comment by ser** [comment]
I can confirm that I have tested WS and it is working correctly now. Thanks again for help, @SomberNight
